documentation/enterprise/enterprise-distribution.md

```diff
@@ -0,0 +1,312 @@
+
+{% endcapture %}
+{% include modules/one-column.html
+  id="sideloading"
+  content=content
+  aside=""
+%}
+
+<!-- END: Single Column Body Module -->
+
+<!-- Single Column Body Module -->
+
+{% capture content %}
+
+## Installation using the Windows registry
+
+This section explains how to install add-ons into Firefox using the Windows Registry.
+
+<!-- Alert -->
+
+{% capture alert %}
+
+Before Firefox 62 it was possible to load unpacked extensions by making the Windows registry key point to a directory containing an unpackaged extension.
+
+From Firefox 62 this is no longer possible, and the key must point to a packaged XPI file, as described in this section.
+
+{% endcapture %}
+{% include modules/note.html
+	content=alert
+	alert=true
+%}
+
+<!-- END: Alert -->
+
+<!-- Note -->
+
+{% capture note %}
+
+It is safe to modify the Registry keys while Firefox is running.
+
+{% endcapture %}
+{% include modules/note.html
+	content=note
+	alert=false
+%}
+
+<!-- END: Note -->
+
+1. Ensure the add-on has an add-on ID, by including the following to its manifest.json file, replacing _your-add-on-name@your-domain.com_ with a suitable ID for your add-on:
+
+<!-- Syntax Highlighting -->
+
+{% highlight javascript linenos %}
+"applications": {
+"gecko": {
+"id": "your-add-on-name@your-domain.com"
+}
+}
+{% endhighlight %}
+
+<!-- END: Syntax Highlighting -->
+
+An email address style ID is recommended.
+
+2. Sign your add-on on addons.mozilla.org (AMO) using the unlisted option. For more details, see Signing and distributing your add-on.
+
+3. Download the signed XPI file and ensure the file name is the add-on ID plus the extension .xpi. For example, `c:/webext/borderify@example.com.xpi`
+
+4. Open Regedit and add keys as follows:
+
+   - For all users of the computer, add to the following registry keys:
+     `HKEY_LOCAL_MACHINE\Software\Mozilla\Firefox\Extensions`
+
+   or
+
+   `HKEY_LOCAL_MACHINE\Software\Wow6432Node\Mozilla\Firefox\Extensions`
+
+   <!-- Note -->
+
+   {% capture note %}
+
+   `HKEY_LOCAL_MACHINE\Software\Mozilla\Firefox\Extensions` is not available when running 32-bit Firefox on a 64-bit machine, you can only install for all users using the `Wow6432Node` key.
+
+   {% endcapture %}
+   {% include modules/note.html
+   	content=note
+   	alert=false
+   %}
+
+   <!-- END: Note -->
+
+- For the current user, add to the following registry key:
+
+`HKEY_CURRENT_USER\Software\Mozilla\Firefox\Extensions`
+
+5. Create a new string value Registry entry with its name equal to the add-on ID, for example, borderify@example.com, and a value equal to the location where the add-on is stored, for example, `c:/webext/borderify@example.com.xpi`.
+
+6. Restart Firefox. The add-on is detected, but the user may be presented with an interstitial or need to enable the add-on in Add-on manager before it can be used. See Firefox settings.
```

Comment:
link to Firefox settings
```suggestion
6. Restart Firefox. The add-on is detected, but the user may be presented with an interstitial or need to enable the add-on in Add-on manager before it can be used. See [Firefox settings](/documentation/enterprise/enterprise-distribution#firefox-settings.
```